apmpackage/apm/changelog.yml

```diff
@@ -3,6 +3,9 @@
     - description: Placeholder
```

Comment:
Should the placeholder be removed manually, or do we remove it when the release happens?

Reply:
Afair, it's removed when we branch out.